src/main/scala/com/raphtory/core/storage/pojograph/PojoGraphLens.scala

```diff
@@ -111,11 +114,13 @@ final case class PojoGraphLens(
   ): Unit = {
     vertices.foreach { case (id, vertex) => f(vertex, graphState) }
     vertexCount.set(vertices.size)
+    StorageTelemetry.pojoLensVertexCount.inc()
```

Comment:
I think with this being outside of the foreach it is just counting the number of functions run? What do you want it to count?